schulcloud.xcodeproj/project.pbxproj

```diff
@@ -808,7 +808,7 @@
 						ProvisioningStyle = Automatic;
 						SystemCapabilities = {
 							com.apple.ApplicationGroups.iOS = {
-								enabled = 0;
+								enabled = 1;
```

Comment:
Also FileProvider related?

Reply:
Yes, you can only communicate via the shared container. So I use the shared container right now to store the downloaded files. Since i was reworking that I just stored the downloaded files there.